Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -39,9 +41,40 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def find_conflict_pr(cls, remote, branch):
-        for pr in remote.pull_requests.find(head=branch, opened=True):
-            return pr
+    def radar_for_id(cls, radar_id):
+        for tracker in Tracker._trackers:
+            if isinstance(tracker, radar.Tracker):
+                return tracker().client.radar_for_id(radar_id, additional_fields=['sourceChanges'])
+
+    @classmethod
+    def find_conflict_pr(cls, remote, radar_id):
+        """
+        Because we don't know what the target branch was just given the radar id,
+        we need to search for PRs with branches that start with the integration prefix
+        and have the first and last sha.
+        """
+        radar_obj = cls.radar_for_id(radar_id)
+        assert radar_obj, 'Could not fetch radar object for id {}'.format(radar_id)
+        shas = []
+
+        repo_name = remote.name if '/' not in remote.name else remote.name.split('/', 1)[-1]
+        for entry in radar_obj.sourceChanges.splitlines():
```

Comment:
I'll let @briannafan chime in, but this sounds like a case where we should be abstracting "sourceChanges" on issue trackers more generally. I can see a world where, for example, we feed this same command a Bugzilla instead.

At the moment, the best implementation of this probably involves just scraping Bugzilla comments for commit annotations. Also, this is more accurately returning refs, I think. It seems possible, for example, that we would get an identifier instead of a hash. The below logic should handle that just fine since you're basically looking for a PR that matches any 1 of an enumerated pattern.

Reply:
While this is in open source code, the conflict workflow is intended for internal users. Do you think it makes sense to work that into this PR, or should we proceed with as-is for now and reevaluate if needed at a later time?

Reply:
Supporting `sourceChanges` for bugzilla would require significant changes - in the spirit of having this tool available, I think this can come at a later time. That said, we can clean up the code here to use the `Issue` class to make that transition easier (see my other comments)